vmupdate/agent/source/common/package_manager.py

```diff
@@ -276,7 +276,10 @@ def upgrade_internal(self, remove_obsolete: bool) -> ProcessResult:
         Just run upgrade via CLI.
         """
         cmd = [self.package_manager,
-               "--noconfirm" if self.package_manager == "pacman" else "-y",
+               *(
```

Comment:
i just extended the existing pattern but please let me know if you'd prefer I clean this up in some way. as-is package manager specifics are leaking into this common module.

Reply:
This is already fixed in https://github.com/QubesOS/qubes-core-admin-linux/pull/164, but it waits for resolving conflicts

Reply:
FYI the other PR is merged now, you can rebase to resolve the conflict.